x/twasm/keeper/genesis.go

```diff
@@ -129,12 +131,27 @@ func ExportGenesis(ctx sdk.Context, keeper *Keeper) *types.GenesisState {
 		contracts[i].ContractState = &types.Contract_CustomModel{CustomModel: &types.CustomModel{Msg: got}}
 	}
 
+	var privileged []string
```

Comment:
"pinned" code and "privileged" contract meta data are also stored in the CodeInfo and ContractDetails. 
After thinking about this a bit more, we should not fill the pinned/privileged field here in export. This would duplicate the information and may cause issues when humans modify them for existing codes/ contracts.

In "seed" mode we don't have these fields explicitly set in the genesis for PoE contracts. They can be used though for other codes/ contracts that are not pinned/ privileged yet.  Maybe some better naming can help?